osu.Game/Localisation/GlobalActionKeyBindingStrings.cs

```diff
@@ -129,6 +129,16 @@ public static class GlobalActionKeyBindingStrings
         /// </summary>
         public static LocalisableString SelectNext => new TranslatableString(getKey(@"select_next"), @"Next selection");
 
+        /// <summary>
+        /// "Previous group selection"
+        /// </summary>
+        public static LocalisableString SelectPreviousGroup => new TranslatableString(getKey(@"select_previous_group"), @"Previous group selection");
```

Comment:
Might just be me but I think "Select previous group" aka the property name reads better than the localisation (maybe the same goes for the two above).

Reply:
Can agree, right now it feels disconnected from all the other settings (noun rather than verb).

Reply:
I'm going to change just these for now. Let's see what sticks going forwards.